tests/integration_test/run_algorithm.py

```diff
@@ -0,0 +1,123 @@
+# Util imports
+from input_output.file_output import load_graph_list_from_filepath
+from input_output.sys_output import passed, fail
+import sys
+import tkinter as tk
+from tkinter import filedialog
+from time import time
+from tests.integration_test.tournament_output import tournament_output
+from tests.integration_test.test_output import test_output
+# Algorithm imports
+from tests.integration_test.settings import *
+from tests.integration_test.isomorphism_problem import are_isomorph, amount_of_isomorphisms
+
+"""
+General integration test for the Graph Isomorphisms problem.
+Settings of the algorithm must be changed in > settings.py <
+"""
+
+"""
+FILE INPUT
+"""
+# Select files to evaluate
+root = tk.Tk()
+root.withdraw()
+file_paths = filedialog.askopenfilenames()
+
+"""
+RUN ALGORITHM
+"""
+if run_mode == 1:
+    error_count = 0
+for file_path in file_paths:
+    graphs = load_graph_list_from_filepath(file_path)
+    filename = (file_path.split("/")[-1]).split(".")[0]
+
+    # Some data structures that are used to determine if graphs are isomorphic more efficiently
+    isomorphisms = []       # List of lists that saves all isomorphic pairs (or more than 2, if that is the case)
+    iso_count = {}          # Dictionary that saves for each graph the amount of automorphisms
```

Comment:
Ik vind de naamgeving een beetje verwarrend. Misschien een idee om de counts allemaal auto_count te noemen. Omdat we eigenlijk nergens meer het aantal isomorfismen berekenen, maar alleen het aantal automorfismen van grafen.

Reply:
goed idee

Reply:
Ik heb ze in de output files nog gewoon iso en iso_count gelaten

Reply:
Dat is prima, daar kijk je zelf toch minder in.